docs/source/example_notebooks/gcm_linear_scm_bootstap_pvalues/false_discovery_rate_control.py

```diff
@@ -0,0 +1,152 @@
+"""Functions in this module should be considered experimental, meaning there might be breaking API changes in the
+future.
+"""
+
+from typing import Union, List, Optional, Callable
+from scipy.stats import rankdata
+import numpy as np 
+
+def numberfy(x):
+    ''' if x could be a number return it as numeric even if it is in string format '''
+    try:
+        nx=float(x)
+    except:
+        nx=np.NaN
+    return(nx)
+
+def adjusted_FDR_pvalues(unajusted_p_values: Union[np.ndarray, List[float]]) -> np.ndarray:
```

Comment:
Is the only difference between this implementation and the one in statsmodels the support of `nan` and float-string values? If this is the case, I am not sure if we really need another re-implementation of it, one can simply make a preprocessing of the values. For instance:
```
p_values = np.array([0.05, 0.01, np.nan, 0.1])
multipletests(pvals=p_values[~np.isnan(p_values)], method='fdr_bh')[1]
```

Reply:
I agree - I needed this to handle the output from semopy - but now I am fully converted to using DoWhy /SCM